src/plugins/data/common/search/aggs/agg_type.ts

```diff
@@ -224,6 +225,8 @@ export class AggType<
     return false;
   }
 
+  getResponseId: (agg: TAggConfig) => string;
```

Comment:
Would be great to add a comment (document) this one explaining the current use-case as an example

Reply:
This also should help with increased `Public APIs missing comments` stat